src/library/f3d_options.h

```diff
@@ -15,6 +17,17 @@ class F3D_EXPORT options
 public:
   options();
   ~options();
+  options(const options& opt);
+  options& operator=(const options& opt);
+
+  class options_exception : public std::runtime_error
```

Comment:
what about just `exception` since it's already in the `options` scope?  
I think `catch(f3d::options::exception)` is cleaner than `catch(f3d::options::options_exception)`